distill/sessions/sessions.py

```diff
@@ -0,0 +1,100 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import distill
+from enum import Enum
+import json
+
+class Sessions_Type(Enum):
+    TAB = "tab"
+    DOMAIN = "domain"
+    DEFAULT = "default"
+
+class Sessions:
+    """
+    A collection of Session objects.
+    """
+
+    ##############################################
+    # IMPLEMENTATION OF PYTHON DEFINED FUNCTIONS #
+    ##############################################
+
+    def __init__(self, sessions=[]):
```

Comment:
We should move the `chunk_by_user_sessions` logic into this class since that method takes in logs and returns a Sessions object.